### ISSUE TITLE
Not clear what's the `slug`

### ISSUE THREAD

**Issue by ayushkovskiy** [neuromation/cookiecutter-neuro-project#67]
During the installation, cookiecutter asks:
```
email [Your email address (e.g. you@example.com)]: 
project_name [Name of the project]: 
project_slug [name-of-the-project]: 
project_short_description [A short description of the project]:
...
```
From the user's point of view, it's unclear what's the `project_slug` and what's the difference between it and `project_name`. We need to add a better description, for example: `project_slug [ID of the project. Can contain only alphanumeric characters and dashes. Default: name-of-the-project]`

**Comment by mariyadavydova** [comment]
Let's rename project_slug to repo_name, it is much more descriptive.

**Comment by anayden** [comment]
I assume #88 covers this one as well? Shall we close this one?

**Comment by mariyadavydova** [comment]
I will fix this in the scope of #66.
